src/sbt-test/sbt-scalafix/basic/build.sbt

```diff
@@ -38,7 +36,7 @@ checkLogs := {
     .map(_.replaceAll("\u001B\\[[;\\d]*m", "")) // remove control chars (colors)
     .force
   assert(
-    logLines.exists(_ == "[error] -import scala.concurrent.Future"),
+    logLines.exists(_.startsWith("[error] -import scala.concurrent.Future")),
```

Comment:
for some weird reason, the lines have non-visible trailing characters when the scripted is run from sbt2 in JDK<21